lib/helpers.js

```diff
@@ -60,6 +60,24 @@ exports.checkPropertyName = function(name) {
 };
 
 /**
+ * Validate remote method name. Allows `.` in method name for prototype
+ * @param {String} name The user input
+ * @returns {String|Boolean}
+ */
+exports.checkRemoteMethodName = function(name) {
```

Comment:
Please keep the naming consistent and rename this function to `validateRemoteMethodName`.